hw/top_earlgrey/ip/xbar_main/dv/autogen/xbar_main_sim_cfg.hjson

```diff
@@ -12,9 +12,6 @@
   // Testplan hjson file.
   testplan: "{proj_root}/hw/top_earlgrey/ip/{dut}/data/autogen/{dut}_testplan.hjson"
 
-  // Add xbar_main specific exclusion files.
```

Comment:
How are the auto-generated cover cfg and el file set in the HJson?

Reply:
Good catch. I accidentally removed this when resolving the conflict. Updated, PTAL